prepare-commit-msg

```diff
@@ -5,11 +5,16 @@ set -eu
 append_coauthors() {
     COMMIT_MSG_FILE=$1
 
-    tmpfile="$COMMIT_MSG_FILE"-temp
-    sed -e '/^Co-authored-by/d' $COMMIT_MSG_FILE >"$tmpfile"
-    mv "$tmpfile" "$COMMIT_MSG_FILE"
+    if grep -q -i -e "Co-authored-by" "$COMMIT_MSG_FILE"; then
+      echo 'Co-author already supplied'
```

Comment:
How do we update the "Co-authored-by" then?

Reply:
It does not try to update the `co-authored-by`.  It simply tries to avoid the case where the `git-pair` script would add extra `co-authored-by` tags.  The assumption being that the user is providing the tag, and they know what they are doing.

Reply:
My point is that this change makes it impossible to update the `Co-authored-by` lines. If I run `git comment --amend` then the `Co-authored-by` lines should be updated according to the currently set co-authors. This change prevents that.